pkgs/applications/networking/netclient/default.nix

```diff
@@ -0,0 +1,34 @@
+{ buildGoModule
+, fetchFromGitHub
+, lib
+, libX11
+, stdenv
+, darwin
+}:
+
+buildGoModule rec {
+  pname = "netclient";
+  version = "0.21.0";
+
+  src = fetchFromGitHub {
+    owner = "gravitl";
+    repo = "netclient";
+    rev = "v${version}";
+    hash = "sha256-68/BmVoAFaIg4vgjzhedSBqm6H9VDu3M7JemfPEcpjQ=";
+  };
+
+  vendorHash = "sha256-CsW4tW6+INw93A7uXtHeVnxRrE5unHXhm2SOmQkJwYA=";
+
+  buildInputs = [ ] ++ lib.optional stdenv.isDarwin darwin.apple_sdk.frameworks.Cocoa
```

Comment:
```suggestion
  buildInputs = lib.optional stdenv.isDarwin darwin.apple_sdk.frameworks.Cocoa
```

Reply:
Thought this would be your suggestion. Anyway it's done